ui/src/components/Footer.tsx

```diff
@@ -62,6 +78,7 @@ export function Footer() {
             >
               TxnLab/reti
```

Comment:
The disclaimer link can actually replace the "TxnLab/reti" link. There's already a link to the GitHub repo in the footer and the org here is no longer accurate.